docs/scripts/gen-faq.js

```diff
@@ -0,0 +1,151 @@
+// read.js
+const fs = require('fs')
+const yaml = require('js-yaml')
+const { Remarkable } = require('remarkable')
+const path = require('path')
+const yamlPath = path.resolve('./docs/faq.yaml')
+
+// Generating FAQ.mdx
+
+if (!fs.existsSync(yamlPath)) {
+  //file exists
+  console.warn('.yaml File does not exists, skipping generating FAQ')
+  return 0
+}
+
+let faqYaml = fs.readFileSync(yamlPath, 'utf8')
+let faq = yaml.load(faqYaml)
+
+const tags = Array.from(new Set(faq.map(({ tags }) => tags).flat(1)))
+
+let data = `---
+id: faq
+title: Frequently Asked Questions (FAQ)
+---
+
```

Comment:
```suggestion

<!-- This file is generated. Please edit /docs/faq.yaml or /docs/scripts/gen-faq.js instead. Changes will be overwritten otherwise -->
```